### DISCUSSION TITLE
Using escape to close expanded page search

### DISCUSSION THREAD

**Discussion by Fred-Vatin** [hoothin/SearchJumper#181] [Q&A] [answered]
Firefox | OS              | Extension
------- | --------------- | ---- 
131.0.2 | Win 10 pro 22H2 | 1.9.3.1

Hello,

I use `ALT+S` to open the find bar. I found out that I can press this shortcut until I lost focus then I can press `escape` to close. Nice !

Unfortunately, I use `ALT+A` to open the expanded page search but didn’t find a way to close it with `escape`. Any solution ? I need to use the mouse and click outside.

**Comment by hoothin** [comment, marked as answer]
Just need to update to the latest version.

**Reply by Fred-Vatin** [reply]
1.9.3.2 allows now to escape the find bar but `escape` still doesn’t work on expanded page. Tested on Firefox.

**Reply by hoothin** [reply]
Sorry, I can't reproduce, any video?

https://github.com/user-attachments/assets/67fa0b1f-ebcc-4764-9661-3fbda9316ce0

**Reply by Fred-Vatin** [reply]
I found the culprit: `Vimimum C` extension steal the `escape` event while the search page is open (I guess). I’m investigating if I can fix it because I really love Vimium as a tool to use keyboard to do various actions on the page and as a very fast way to type a search term and send it to any engine when adding a keyword. Only via keyboard.

**Reply by hoothin** [reply]
I guess key `i` may be helpful under Vimimum.
